server/db/migration/V20241008.1761__add_projectShare_test.sql

```diff
@@ -0,0 +1,9 @@
+INSERT INTO dbo.ProjectShare (email, projectId)
```

Comment:
I forgot to remove this but i'll fix it after any comments

Reply:
I removed that migration, and pushed the change up to git hub.